x/params/legacy/v044/store.go

```diff
@@ -0,0 +1,20 @@
+package v043
+
```

Comment:
We should also add a `json.go` to do JSON genesis migrations.

Btw, where do you envision this param going, in an exported genesis JSON?

Reply:
this is already in the genesis as its part of tendermints params, it was never used in the Cosmo-sdk

Reply:
> Are consensus params exported in the sdk?

Ah, no, let's keep them the tm params then. Should we add an Initgenesis though?

Reply:
> Should we add an Initgenesis though?

Just saw that `StoreConsensusParams` is called in InitChain, so no need for Initgenesis.